src/ParameterParser.h

```diff
@@ -403,6 +403,32 @@ namespace snowcrash {
             }
         }
 
+        static size_t matchBracket(const mdp::ByteBuffer& text) {
```

Comment:
There should already be a function like this somewhere in snowcrash. We can use that.